src/jira.py

```diff
@@ -0,0 +1,144 @@
+import base64
+import re
+import requests
+import pprint
+import time
+
+from src.access import get_access_params
+from src.board import Issue
+
+
+class JiraBoard:
+
+    def __init__(self, project_key):
```

Comment:
You could define `project_key: str` here (https://docs.python.org/3.6/library/typing.html), that would way type information does not need to be included in the docstring.

Reply:
Done. Is it better to never use docstrings at all?

Reply:
No, docstrings are great and absolutely needed. But with respect to bevity (i.e., developers reading your code) it would be more concise to include type information in the suggested way.